dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -1092,15 +1100,17 @@ private void unexportMetadataService() {
     private void exportServices() {
         configManager.getServices().forEach(sc -> {
             // TODO, compatible with ServiceConfig.export()
-            ServiceConfig serviceConfig = (ServiceConfig) sc;
+            ServiceConfig<?> serviceConfig = (ServiceConfig<?>) sc;
             serviceConfig.setBootstrap(this);
 
-            if (exportAsync) {
+            ApplicationConfig config = getApplication();
+            if (config.getExportAsync() != null && config.getExportAsync()) {
```

Comment:
Why not design the `exportServices` to be asynchronous instead of relying on `export-async` parameter?

Reply:
I think it's better to let user to choose.

Reply:
In this case, we know that `async` is better than `sync`, why don't we recomend `async` or set it as default?

Reply:
fair point. I'll change it.